docs/themes/code_mode.rst

```diff
@@ -1,6 +1,52 @@
 .. vale off
 
-Code Mode
+Code mode
 ##########
 
-.. vale on
\ No newline at end of file
+.. vale on
+
+Code Mode is an option available in the Email and Landing Page edit Form. It allows you to create/insert/edit your content in HTML code. It's helpful for situations where you don't want to use a Mautic Theme and you want to use an HTML Theme copied from a ``3rd`` party Theme builder or if you enjoy editing HTML code so much.
+
+With Mautic 2.3.0, Code Mode replaced the full-page Froala (WYSIWYG) editor that sometimes altered the HTML code. Code Mode won't modify any of the code you paste in. The other option to edit a page/email content is to use the Builder. It uses Froala editor only to edit the text/image content, not the full page, so that it won't modify the page layout.
+
+Select the Code mode
+********************
+
+After creating or editing the page/email, you can select the Code Mode from the Theme selector. To open the Code Mode Builder, click the Builder button after selecting the Code Mode Theme option.
```

Comment:
```suggestion
After creating or editing the Landing Page/Email, you can select code mode from the Theme selector. To open the code mode Builder, click the advanced tab which appears.
``` 

Please add a screenshot here to show how to work with the code mode builder (under the Advanced tab - this is new since 4.2 I think it was).